docs/src/components/Typography/type-scale.scss

```diff
@@ -41,10 +46,4 @@
       color: $bpk-color-sky-gray-tint-03;
     }
   }
-
-  &__table {
```

Comment:
Removed as we no longer want to be overly complex and fancy. Something we should've originally pushed back on